Framework/Algorithms/test/MaxEntTest.h

```diff
@@ -313,7 +308,6 @@ class MaxEntTest : public CxxTest::TestSuite {
     alg->setChild(true);
     alg->setProperty("InputWorkspace", ws);
     alg->setProperty("A", 0.01);
-    alg->setProperty("ChiTarget", 50.);
```

Comment:
should we not be setting a smaller `ChiTarget` value e.g. 0.5?

Reply:
What information would it give us? We are testing the resolution factor here.

Reply:
based on the new definition of chiTarget setting it to 50 means that we are using 50*N (N = number of data points). This just seemed a bit excessive to me

Reply:
This line is deleted. No ChiTarget is set.